randovania/games/dread/json_data/Dairon.txt

```diff
@@ -143,7 +154,13 @@ Extra - asset_id: collision_camera_001
   > Top Floor
       After Dairon - Teleport To Artaria Speed Blocks Destroyed
   > Event - Speed Booster blocks Destroyed
-      Speed Booster
+      All of the following:
+          Speed Booster
+          Any of the following:
+              Grapple Beam or Spider Magnet or Space Jump or Simple IBJ
```

Comment:
If you climb with spider magnet without walljumping, can you still make this spark in time?

Reply:
there's a slope you can spark into from the wall, so yes